library-app/src/App.module.css

```diff
@@ -0,0 +1,8 @@
+@media only screen and (min-width: 426px) {
+  .wrapp {
+    display: flex;
+    flex-direction: row;
+    /* overflow: scroll; */
+    height: 100vh;
```

Comment:
This is full viewport height only on desktops?

Reply:
on both, mobile and desktops